var/spack/repos/builtin/packages/cbtf-argonavis/package.py

```diff
@@ -58,7 +58,7 @@ class CbtfArgonavis(Package):
             git='https://github.com/OpenSpeedShop/cbtf-argonavis.git')
 
     depends_on("cmake@3.0.2:", type='build')
-    depends_on("boost@1.50.0:")
+    depends_on("boost@1.50.0:boost@1.59.0")
```

Comment:
This should be `1.50.0:1.59.0`. You should also be able to abbreviate them as `1.50:1.59` if you want.